lib/riemann.js

```diff
@@ -113,11 +129,17 @@ Riemann.prototype.handlePacketEvent = function (eventString) {
 Riemann.prototype.send = function (eventData) {
   var eventToSend = this.client.Event(eventData);
 
-  // Send using proper transport
-  if (this.config.transport && this.config.transport === "tcp") {
-    this.client.send(eventToSend, this.client.tcp);
-  } else {
-    this.client.send(eventToSend);
+  try {
+    // Send using proper transport
+    if (this.config.transport && this.config.transport === "tcp") {
+      this.client.send(eventToSend, this.client.tcp);
+    } else {
+      this.client.send(eventToSend);
+    }
+  } catch (err) {
```

Comment:
So does `send` throw then if the TCP socket is closed? Or does `this.client` emit an error?